cmd/adapter-rest/startcmd/start.go

```diff
@@ -134,13 +140,20 @@ func getAdapterRestParameters(cmd *cobra.Command) (*adapterRestParameters, error
 		return nil, err
 	}
 
+	presentationDefinitionsFile, err := cmdutils.GetUserSetVarFromString(cmd, presentationDefinitionsFlagName,
+		presentationDefinitionsEnvKey, false)
+	if err != nil {
+		return nil, err
+	}
+
 	return &adapterRestParameters{
-		hostURL:           hostURL,
-		tlsSystemCertPool: tlsSystemCertPool,
-		tlsCACerts:        tlsCACerts,
-		dbURL:             dbURL,
-		oidcProviderURL:   oidcURL,
-		staticFiles:       staticFiles,
+		hostURL:                     hostURL,
+		tlsSystemCertPool:           tlsSystemCertPool,
+		tlsCACerts:                  tlsCACerts,
+		dbURL:                       dbURL,
+		oidcProviderURL:             oidcURL,
+		staticFiles:                 staticFiles,
+		presentationDefinitionsFile: presentationDefinitionsFile,
```

Comment:
we should start thinking about saving this config to the database, and exposing an endpoint for this configuration.